app/Main.hs

```diff
@@ -49,22 +49,21 @@ usageDoc = Just $ mconcat
 contracts :: ContractRegistry
 contracts = ContractRegistry $ Map.fromList
   [ "BaseDAO" ?:: ContractInfo
-    { ciContract = DAO.baseDaoContract DAO.defaultConfig
+    { ciContract = DAO.baseDaoContract @() @() DAO.defaultConfig
```

Comment:
In another PR I wanted to suggest hiding all these type parameters and config inside the library and exporting the contract with everything applied. We can call it `simplestDAO` for example. Or `trivialDAO`. So that `baseDaoContract` is a generic thing and there is another thing with a different name that is specifically the simplest version of baseDAO.

Reply:
Good, I'd like to try it. Can I apply it right in this PR?

Reply:
Sad that this can't be easily applied to our tests because there we use different origination methods / different configs all the time.

Reply:
> Can I apply it right in this PR?

I don't mind. This PR increases number of things necessary to instantiate a specific contract, so hiding this things is related to the essence of this PR.

> Sad that this can't be easily applied to our tests because there we use different origination methods / different configs all the time.


It shouldn't be a problem if we still export the generic thing (`baseDaoContract`).